vue/wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,137 @@
+<template lang='pug'>
+  .register-container
+    img.wolox-icon(alt='Vue logo' src='../assets/wolox_logo.svg')
+    form.form-container(@submit.prevent="onSubmit()")
+      p.title-form
+        | {{labels.title}}
+      .input-text-container(v-for='(field, index) in fields' :key='index')
+        label.input-text-label(:for='field.name')
+          | {{ field.label }}
+        input.input-text-content(:id='field.name' v-model='user[field.name]' )
+      button.base-form-button
+        | {{ labels.signUp }}
+    .container-button
+      button.base-form-button.login-button
```

Comment:
This is a link

Reply:
Yes but no for this card, on pr the vue-router I change it.